upstream-sha256: b2f74869e259f8572381a210d6cdfea960326dc60d550e3d40d863e0aa1537d2
actual-sha256: f490a2dd8181e23de2e6a2e1dc3e474eb0b5eda0e286b1f51d29155f2746c155
--- upstream.expected
+++ tolk.actual
@@ -3,65 +3,68 @@
    2 Ops.CAST            :            dtypes.int                               ['10']                           dtypes.int
    3 Ops.RANGE           : 1          dtypes.int                               [2]                              (1, AxisType.WEAK)
    4 Ops.INDEX           : 1          dtypes.float                             [0, 3]                           None
-   5 Ops.PARAM           :            dtypes.float                             []                               ParamArg(1, dtypes.float, 100, device='CPU')
-   6 Ops.INDEX           : 1          dtypes.float                             [5, 3]                           None
-   7 Ops.LOAD            : 1          dtypes.float                             [6]                              None
-   8 Ops.ADD             : 1          dtypes.int                               [3, 2]                           None
-   9 Ops.INDEX           : 1          dtypes.float                             [5, 8]                           None
-  10 Ops.LOAD            : 1          dtypes.float                             [9]                              None
-  11 Ops.ADD             : 1          dtypes.float                             [7, 10]                          None
-  12 Ops.CONST           :            dtypes.weakint                           []                               20
-  13 Ops.CAST            :            dtypes.int                               ['20']                           dtypes.int
-  14 Ops.ADD             : 1          dtypes.int                               [3, 13]                          None
-  15 Ops.INDEX           : 1          dtypes.float                             [5, 14]                          None
-  16 Ops.LOAD            : 1          dtypes.float                             [15]                             None
-  17 Ops.ADD             : 1          dtypes.float                             [11, 16]                         None
-  18 Ops.CONST           :            dtypes.weakint                           []                               30
-  19 Ops.CAST            :            dtypes.int                               ['30']                           dtypes.int
-  20 Ops.ADD             : 1          dtypes.int                               [3, 19]                          None
-  21 Ops.INDEX           : 1          dtypes.float                             [5, 20]                          None
-  22 Ops.LOAD            : 1          dtypes.float                             [21]                             None
-  23 Ops.ADD             : 1          dtypes.float                             [17, 22]                         None
-  24 Ops.CONST           :            dtypes.weakint                           []                               40
-  25 Ops.CAST            :            dtypes.int                               ['40']                           dtypes.int
-  26 Ops.ADD             : 1          dtypes.int                               [3, 25]                          None
-  27 Ops.INDEX           : 1          dtypes.float                             [5, 26]                          None
-  28 Ops.LOAD            : 1          dtypes.float                             [27]                             None
-  29 Ops.ADD             : 1          dtypes.float                             [23, 28]                         None
-  30 Ops.CONST           :            dtypes.weakint                           []                               50
-  31 Ops.CAST            :            dtypes.int                               ['50']                           dtypes.int
-  32 Ops.ADD             : 1          dtypes.int                               [3, 31]                          None
-  33 Ops.INDEX           : 1          dtypes.float                             [5, 32]                          None
-  34 Ops.LOAD            : 1          dtypes.float                             [33]                             None
-  35 Ops.ADD             : 1          dtypes.float                             [29, 34]                         None
-  36 Ops.CONST           :            dtypes.weakint                           []                               60
-  37 Ops.CAST            :            dtypes.int                               ['60']                           dtypes.int
-  38 Ops.ADD             : 1          dtypes.int                               [3, 37]                          None
-  39 Ops.INDEX           : 1          dtypes.float                             [5, 38]                          None
-  40 Ops.LOAD            : 1          dtypes.float                             [39]                             None
-  41 Ops.ADD             : 1          dtypes.float                             [35, 40]                         None
-  42 Ops.CONST           :            dtypes.weakint                           []                               70
-  43 Ops.CAST            :            dtypes.int                               ['70']                           dtypes.int
-  44 Ops.ADD             : 1          dtypes.int                               [3, 43]                          None
-  45 Ops.INDEX           : 1          dtypes.float                             [5, 44]                          None
-  46 Ops.LOAD            : 1          dtypes.float                             [45]                             None
-  47 Ops.ADD             : 1          dtypes.float                             [41, 46]                         None
-  48 Ops.CONST           :            dtypes.weakint                           []                               80
-  49 Ops.CAST            :            dtypes.int                               ['80']                           dtypes.int
-  50 Ops.ADD             : 1          dtypes.int                               [3, 49]                          None
-  51 Ops.INDEX           : 1          dtypes.float                             [5, 50]                          None
-  52 Ops.LOAD            : 1          dtypes.float                             [51]                             None
-  53 Ops.ADD             : 1          dtypes.float                             [47, 52]                         None
-  54 Ops.CONST           :            dtypes.weakint                           []                               90
-  55 Ops.CAST            :            dtypes.int                               ['90']                           dtypes.int
-  56 Ops.ADD             : 1          dtypes.int                               [3, 55]                          None
-  57 Ops.INDEX           : 1          dtypes.float                             [5, 56]                          None
-  58 Ops.LOAD            : 1          dtypes.float                             [57]                             None
-  59 Ops.ADD             : 1          dtypes.float                             [53, 58]                         None
-  60 Ops.PARAM           :            dtypes.float                             []                               ParamArg(2, dtypes.float, 10, device='CPU')
-  61 Ops.INDEX           : 1          dtypes.float                             [60, 3]                          None
-  62 Ops.LOAD            : 1          dtypes.float                             [61]                             None
-  63 Ops.ADD             : 1          dtypes.float                             [59, 62]                         None
-  64 Ops.STORE           : 1          dtypes.void                              [4, 63]                          None
-  65 Ops.END             :            dtypes.void                              [64, 3]                          None
-  66 Ops.SINK            :            dtypes.void                              [65]                             KernelInfo(name='r_10_10', applied_opts=(Opt(op=OptOps.SPLIT, axis=1, arg=(0, AxisType.UNROLL)),), opts_to_apply=None, estimates=None, beam=0)
+   5 Ops.CONST           :            dtypes.weakfloat                         []                               0.0
+   6 Ops.CAST            :            dtypes.float                             ['0.0']                          dtypes.float
+   7 Ops.PARAM           :            dtypes.float                             []                               ParamArg(1, dtypes.float, 100, device='CPU')
+   8 Ops.INDEX           : 1          dtypes.float                             [7, 3]                           None
+   9 Ops.LOAD            : 1          dtypes.float                             [8]                              None
+  10 Ops.ADD             : 1          dtypes.float                             [6, 9]                           None
+  11 Ops.ADD             : 1          dtypes.int                               [3, 2]                           None
+  12 Ops.INDEX           : 1          dtypes.float                             [7, 11]                          None
+  13 Ops.LOAD            : 1          dtypes.float                             [12]                             None
+  14 Ops.ADD             : 1          dtypes.float                             [10, 13]                         None
+  15 Ops.CONST           :            dtypes.weakint                           []                               20
+  16 Ops.CAST            :            dtypes.int                               ['20']                           dtypes.int
+  17 Ops.ADD             : 1          dtypes.int                               [3, 16]                          None
+  18 Ops.INDEX           : 1          dtypes.float                             [7, 17]                          None
+  19 Ops.LOAD            : 1          dtypes.float                             [18]                             None
+  20 Ops.ADD             : 1          dtypes.float                             [14, 19]                         None
+  21 Ops.CONST           :            dtypes.weakint                           []                               30
+  22 Ops.CAST            :            dtypes.int                               ['30']                           dtypes.int
+  23 Ops.ADD             : 1          dtypes.int                               [3, 22]                          None
+  24 Ops.INDEX           : 1          dtypes.float                             [7, 23]                          None
+  25 Ops.LOAD            : 1          dtypes.float                             [24]                             None
+  26 Ops.ADD             : 1          dtypes.float                             [20, 25]                         None
+  27 Ops.CONST           :            dtypes.weakint                           []                               40
+  28 Ops.CAST            :            dtypes.int                               ['40']                           dtypes.int
+  29 Ops.ADD             : 1          dtypes.int                               [3, 28]                          None
+  30 Ops.INDEX           : 1          dtypes.float                             [7, 29]                          None
+  31 Ops.LOAD            : 1          dtypes.float                             [30]                             None
+  32 Ops.ADD             : 1          dtypes.float                             [26, 31]                         None
+  33 Ops.CONST           :            dtypes.weakint                           []                               50
+  34 Ops.CAST            :            dtypes.int                               ['50']                           dtypes.int
+  35 Ops.ADD             : 1          dtypes.int                               [3, 34]                          None
+  36 Ops.INDEX           : 1          dtypes.float                             [7, 35]                          None
+  37 Ops.LOAD            : 1          dtypes.float                             [36]                             None
+  38 Ops.ADD             : 1          dtypes.float                             [32, 37]                         None
+  39 Ops.CONST           :            dtypes.weakint                           []                               60
+  40 Ops.CAST            :            dtypes.int                               ['60']                           dtypes.int
+  41 Ops.ADD             : 1          dtypes.int                               [3, 40]                          None
+  42 Ops.INDEX           : 1          dtypes.float                             [7, 41]                          None
+  43 Ops.LOAD            : 1          dtypes.float                             [42]                             None
+  44 Ops.ADD             : 1          dtypes.float                             [38, 43]                         None
+  45 Ops.CONST           :            dtypes.weakint                           []                               70
+  46 Ops.CAST            :            dtypes.int                               ['70']                           dtypes.int
+  47 Ops.ADD             : 1          dtypes.int                               [3, 46]                          None
+  48 Ops.INDEX           : 1          dtypes.float                             [7, 47]                          None
+  49 Ops.LOAD            : 1          dtypes.float                             [48]                             None
+  50 Ops.ADD             : 1          dtypes.float                             [44, 49]                         None
+  51 Ops.CONST           :            dtypes.weakint                           []                               80
+  52 Ops.CAST            :            dtypes.int                               ['80']                           dtypes.int
+  53 Ops.ADD             : 1          dtypes.int                               [3, 52]                          None
+  54 Ops.INDEX           : 1          dtypes.float                             [7, 53]                          None
+  55 Ops.LOAD            : 1          dtypes.float                             [54]                             None
+  56 Ops.ADD             : 1          dtypes.float                             [50, 55]                         None
+  57 Ops.CONST           :            dtypes.weakint                           []                               90
+  58 Ops.CAST            :            dtypes.int                               ['90']                           dtypes.int
+  59 Ops.ADD             : 1          dtypes.int                               [3, 58]                          None
+  60 Ops.INDEX           : 1          dtypes.float                             [7, 59]                          None
+  61 Ops.LOAD            : 1          dtypes.float                             [60]                             None
+  62 Ops.ADD             : 1          dtypes.float                             [56, 61]                         None
+  63 Ops.PARAM           :            dtypes.float                             []                               ParamArg(2, dtypes.float, 10, device='CPU')
+  64 Ops.INDEX           : 1          dtypes.float                             [63, 3]                          None
+  65 Ops.LOAD            : 1          dtypes.float                             [64]                             None
+  66 Ops.ADD             : 1          dtypes.float                             [62, 65]                         None
+  67 Ops.STORE           : 1          dtypes.void                              [4, 66]                          None
+  68 Ops.END             :            dtypes.void                              [67, 3]                          None
+  69 Ops.SINK            :            dtypes.void                              [68]                             KernelInfo(name='r_10_10', applied_opts=(Opt(op=OptOps.SPLIT, axis=1, arg=(0, AxisType.UNROLL)),), opts_to_apply=None, estimates=None, beam=0)
